src/Caliburn.Micro.Core/Caliburn.Micro.Core.csproj

```diff
@@ -1,7 +1,9 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras">
-
+﻿<Project>
+  <Sdk Name="MSBuild.Sdk.Extras" Version="2.0.24" />
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.1;net45;uap10.0.16299;Xamarin.iOS10;MonoAndroid80</TargetFrameworks>
+    <!--<TargetFrameworks>netstandard1.1;net45;uap10.0.16299;Xamarin.iOS10;MonoAndroid80;netcoreapp3.0</TargetFrameworks>-->
+    <TargetFrameworks>net45;Xamarin.iOS10;MonoAndroid80;netstandard2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
Definitely want to keep `netstandard1.1` in this list.

Reply:
My bad I changed that to see if it would resolve the issue I was having that turned out to be the conflict between UAP and coreapp3.0.  I have a bug into msbuild team so we'll see what they say but I will change that back to 1.1.

Reply:
No worries, realize this is still a draft PR. Good work.